### PR TITLE
KEYCLOAK-16957 Update to EAP_736_CR1 and change SSO versions to 7.4.6

### PR DESCRIPTION
**Pull request by ASzc**

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -73,12 +73,12 @@ modules:
...
+                  ref: EAP_736_CR1
 
           - name: jboss-eap-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_735_CR1
+                  ref: EAP_736_CR1
 
           - name: cct_module
             git:
```
Comment (iankko) [inline image.yaml:84]: @ASzc Thanks for the PR! The
```
- name: jboss-eap-7.3.5.GA-CR1-patch
       target: jboss-eap-7.3.5.GA-CR1-patch.zip
       md5: 921892c5a2234ca2bd09d5fd5ad1aad8
       url: http://$DOWNLOAD_SERVER/devel/candidates/jboss/eap/JBEAP-7.3.5.GA-CR1/jboss-eap-7.3.5.GA-CR1-patch.zip
```
section below should be updated to point to ```jboss-eap-7.3.6.GA-CR1-patch.zip``` below yet / instead (bump the artifact to ```jboss-eap-7.3.6.GA-CR1-patch.zip``` where necessary, and update the MD5 sum of the zip as appropriate).

The rest of the changes /lgtm AFAICT.

**Review by iankko — CHANGES_REQUESTED** [review]
Artifact / md5sum updates needed in ```image.yaml``` file yet.

**Review by iankko — CHANGES_REQUESTED** [review]
See previous comment

**Comment by ASzc** [comment]
Ok, thanks @iankko, I have updated the EAP patch referenced in image.yaml

**Comment by iankko** [comment]
> Ok, thanks @iankko, I have updated the EAP patch referenced in image.yaml

This version /lgtm [1], thanks @ASzc !

--
[1]
```
[iankko@localhost tmp]$ diff -s /tmp/alexs_change.diff /tmp/7.4.6-expected-change 
Files /tmp/alexs_change.diff and /tmp/7.4.6-expected-change are identical
```

**Review by iankko — APPROVED** [review]
/lgtm

**Review by pskopek — APPROVED** [review]
looks good!

**Pull request merged**